app/views/visns/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<% provide :body_classes, 'bg-gray-2' %>
+<% provide :footer_classes, 'bg-gray-2' %>
+
+<div id="visns-index" class="grid-container">
+  <%= render partial: "shared/messages", locals: { small_text: false } %>
+  <section id="visns-map" class="margin-bottom-3">
+    <h1 class="font-sans-2xl margin-top-8 margin-bottom-3 line-height-46px">VISN directory</h1>
+    <div class="margin-bottom-10">
+      <span class="line-height-26">Looking for a list of VAMCs?</span> <%= link_to 'Click here', vamcs_path, class: 'dm-external-link' %>
```

Comment:
I would double check with Becca but I think instead of the `dm-external-link` class here we should probably use the `dm-internal-link` class because it is linking to another DM page... I know sometimes we aren't consistent with that and it might be the case here... then please ignore this comment hahah 😅